QuickshaRe/UI/TextInputView.swift

```diff
@@ -15,12 +15,16 @@ struct TextInputView: View {
     
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: QuickshaRe/UI/TextInputView.swift;
  Line: 15;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)